plugins/nvme/nvme.quirk

```diff
@@ -0,0 +1,3 @@
+# SK hynix
+[DeviceInstanceId=NVME\VID_1C5C]
```

Comment:
Just tested this on this disk.  I found the instance is built as `NVME\VEN_1C5C`, not `NVME\VID_1C5C`.  I think you actually want the later though right so this is consistent between usb and udev?  It looks like fu-udev-device is also using DEV instead of PID

Reply:
Gahh, you're right. Microsoft specify VEN&DEV for the device-id -- PCI/VID&PID would never exist. I'll fix up the quirk file now.